CHANGELOG.md

```diff
@@ -27,6 +27,10 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - `to_tensor` now also handles `numpy` arrays
 - `MIN` mode of `NumericalTarget` is now implemented via the acquisition function
   instead of negating the computational representation
+- Fingerprint encodings are now computed exclusively with `scikit-fingerprints` package, 
+  granting access to all fingerprints available within `scikit-fingerprints`.
+  The `SubstanceParameter` value `MORGAN_FP` is now an alias for `ECFP` 
```

Comment:
please put the sentence describing the deprecation under `Deprecations`, see previous changelog entries for examples